src/components/EditionCard/EditionCard.test.tsx

```diff
@@ -88,3 +91,40 @@ describe("Edition Year with Minimal Data", () => {
     expect(screen.queryByText("Read Online")).not.toBeInTheDocument();
   });
 });
+
+describe("Edition with EDD", () => {
```

Comment:
Add additional tests for Edition with both EDD and regular read-online/download (should show read-online/download)

Reply:
These test names are a bit difficult to read and parse.  
Can we say, for each test, what the data is that you're checking, and what you expect?  
"Shows Download and Read Online button when edition has both EDD and readable links", 
"Shows Login button when EDD is available but user is not logged in", etc etc etc

Reply:
Updated, thanks!